### PR TITLE
minor additions

### PR DESCRIPTION
**Pull request by ComputerScientist-01**
feel free to see my solutions

### CODE REVIEW

File: Jayvardhan/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+</p>
+
+<br>
+
+## :man: Repository Created & Maintained By -
+
+- **Hey guys, I'm Jayvardhan. Find out more about me** [ here](https://linkedin.com/in/rathi406)
+- **Reach out to me at** [rathi406@gmail.com](rathi406@gmail.com)
+
+<h3 align="right">Built with :heart: by Jayvardhan Rathi</h3>
```
Comment (AdityaPrakash-26) [inline Jayvardhan/README.md]: Please undo this change

**Review by AdityaPrakash-26 — CHANGES_REQUESTED** [review]
Other than the `README.md`, everything LGTM!

File: Jayvardhan/README.md (hunk 1 above)
Comment (ComputerScientist-01) [reply Jayvardhan/README.md]: readme has been removed

**Comment by AdityaPrakash-26** [comment]
LGTM!

**Pull request merged**